ribs/archives/_cvt_archive.py

```diff
@@ -118,6 +118,8 @@ def __init__(self,
                  dtype=np.float64,
                  samples=100_000,
                  custom_centroids=None,
+                 chunk_size=10,
```

Comment:
I think you can just have this default to None. If someone passes in a chunk size, e.g., 500, then we will use it. It's a rare enough operation that this doesn't need to have a default value of 10.